### DISCUSSION TITLE
pikpak网盘挂载提示An existing connection was forcibly closed by the remote host.

### DISCUSSION THREAD

**Discussion by linger6580** [AlistGo/alist#6132] [Q&A]
alist挂载pikpak网盘，状态提示work，但是点进文件夹即提示如下信息：
failed get objs: failed to list objs: Get "https://api-drive.mypikpak.com/drive/v1/files?filters=%7B%22phase%22%3A%7B%22eq%22%3A%22PHASE_TYPE_COMPLETE%22%7D%2C%22trashed%22%3A%7B%22eq%22%3Afalse%7D%7D&limit=100&page_token=&parent_id=&thumbnail_size=SIZE_LARGE&with_audit=true": read tcp ***********1:42702->***********7:443: wsarecv: An existing connection was forcibly closed by the remote host.
求帮助！

**Comment by SyberRabbit** [comment]
应该是被墙吧，我也不清楚。
我这里的情况是，国内网提示强制关闭连接，而新加坡的云服务器，就可以加上
